editor/cypress/fixtures/titanic.json

```diff
@@ -30,6 +28,28 @@
     "parentField": "ml:parentField",
     "path": "ml:path",
     "personalSensitiveInformation": "ml:personalSensitiveInformation",
+    "dataBiases": "ml:dataBiases",
```

Comment:
Can you respect the alphabetical order in the `@context`?

You should probably use the [`@context`](https://github.com/mlcommons/croissant/blob/7ce531d0162eb2fcbe2891fbd6a68dc71b60af6c/python/mlcroissant/mlcroissant/_src/core/rdf.py#L19-L58) and do [a proper migration](https://github.com/mlcommons/croissant/blob/main/python/mlcroissant/mlcroissant/scripts/migrations/migrate.py) to migrate all existing JSON-LD files. However, I'm not sure we want to always have the whole context... I can discuss this with Omar and Pierre.

Reply:
3 tests remain, but I'm not figuring out why the Notebook Tests are failing. Can you help me in that sense? 
About formatting, I'm now using the correct black version, 23.11.0 (I was using 24.x…), but I'm still not passing the test because metadat.py file. :(

On the other side, maybe, I should wait for a conclusion of RAI in the context before migrating all JSON-LD files.